README.md

```diff
@@ -44,9 +44,9 @@ Create a JSON object with required data, then use it in such cases:
   "token": "YOUR_TOKEN",
   // For embedded application only. Events also should be dispatched or listened in the root application.
   "customEvents": {
-    "TRIGGER_MODAL": "triggermodal", // Root application can dispatch this event to open a prompt in MARVA which will inform a user about unsaved changes before leaving an Edit or Create page.
-    "PROCEED_NAVIGATION": "proceednavigation", // MARVA dispatches this event when a user clicks in the prompt "Save and continue" button or closes the prompt.
-    "BLOCK_NAVIGATION": "blocknavigation" // MARVA dispatches this event when user makes changes in a work form ("Create" or "Edit" page).
+    "TRIGGER_MODAL": "triggermodal", // Root application can dispatch this event to open a prompt in Linked data which will inform a user about unsaved changes before leaving an Edit or Create page.
```

Comment:
Instead of adding the documentation as comments, please add  a paragraph (or a table) above this explaining what is meant  by each of these configs.